test/pytest/test_rnn.py

```diff
@@ -1,13 +1,11 @@
 import pytest
 import hls4ml
-import tensorflow as tf
 import numpy as np
 from pathlib import Path
-from tensorflow.keras import optimizers
-from tensorflow.keras.models import Model
-from tensorflow.keras.layers import Input, Embedding, SimpleRNN, LSTM, GRU
-import math
-from tensorflow.keras import backend as K
+
+from traitlets import default
```

Comment:
Auto-import issue?

Reply:
Yeah, not sure where these came from... Reverting